ex03.exs

```diff
@@ -60,7 +60,29 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    « your code here »
+    collection
+    |> splitter(process_count)
+    |> Enum.map(fn chunk -> make_task(chunk, function) end)
+    |> Enum.map(fn task -> Task.await(task) end)
+    |> Enum.concat()
+  end
+
+  # HELPER FUNCTIONS
+
+  defp make_task(list, function) do
+    Task.async(fn -> Enum.map(list, function) end)
+  end
+
+
+  # Splits enumerable into n chunks as evenly as possible
```

Comment:
This is novel. Most people either used `Enum.chunk_every` or just plain got it wrong. Your approach is actually an implementation of `chunk_every`.